components/Input.html

```diff
@@ -36,7 +36,89 @@
                 </ul>
             </aside>
             <div class="main__grid-item">
-                
+                <div class="component">
+                    <p class="component__heading">Input fields</p>
+                    <p class="component__text">Inputs are necessary for user interaction with the 
+                        website. Inputs are used to take information from the user, and then save the 
+                        data to the server, accordingly.</p>
+                    <p class="component__heading-primary">Radio Inputs</p>
+                    <p class="component__sub-text">
+                        Radio buttons allow the user to select one option from a set.
+                    </p> 
+                    <div class="component__view">
+                         <div class="radio">
+                             <label for="radio-1">
+                                 <input type="radio" name="radio" class="radio__input" checked/>
+                             </label>
+                             Checked
+                         </div>
+                         <div class="radio">
+                            <label for="radio-2">
+                                <input type="radio" name="radio" class="radio__input"/>
+                            </label>
+                            Unchecked
+                        </div>
+                        <div class="radio">
+                            <label for="radio-1">
+                                <input type="radio" name="radio" class="radio__input" disabled/>
+                            </label>
```

Comment:
incorrect syntax
```
<label for="radio-1">Selected</label>
<input type="radio" id="radio-1" name="radio-group-1">
```
`name`  is used to group multiple radio buttons. `id` is used to identify individual radio buttons. `for` should match the unique `id` to work properly. Don't wrap the `input` inside the `for`